### PR TITLE
Dba 764

### PR DESCRIPTION
**Pull request by bill-buchan**

### CODE REVIEW

**Review by ranbeersingh1 — DISMISSED** [review]
LGTM

**Review by ranbeersingh1 — DISMISSED** [review]
LGTM

File: playbooks/oracle_audit_replication_management/oracle_audit_replication_management/tasks/wait_for_zero_throughput.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+              --region {{ region }} \
+             --namespace 'AWS/DMS' --period 60 --statistics Maximum \
+             --dimensions Name=ReplicationInstanceIdentifier,Value={{ replication_instance_identifier }} \
+                          Name=ReplicationTaskIdentifier,Value={{ audited_interaction_task_arn.split(':')[-1] }} \
+             --query "Datapoints[0].Maximum" --output text
+  register: get_cdc_throughput_row_source
+  until: (get_cdc_throughput_row_source.stdout == '0.0') or (get_cdc_throughput_row_source.stdout == 'null') or  (get_cdc_throughput_row_source.stdout == 'None')
+  retries: 20
+  delay: 60
+  delay: 6
```
Comment (georgepstaylor) [inline playbooks/oracle_audit_replication_management/oracle_audit_replication_management/tasks/wait_for_zero_throughput.yml:17]: Formatter check is warning about a duplicate key here @bill-buchan

Comment (bill-buchan) [reply playbooks/oracle_audit_replication_management/oracle_audit_replication_management/tasks/wait_for_zero_throughput.yml]: Oops - now removed

**Review by georgepstaylor — APPROVED** [review]

**Pull request merged**